Assets/Source/Entities/Behaviors/Commander.cs

```diff
@@ -7,9 +7,11 @@ public class Commander : MonoBehaviour
 {
 	// Get rid of these after testing movement
 	private GameObject tempCircle;
+	private Player player;
 
 	void Start()
 	{
+		player = transform.root.GetComponent<Player>();
```

Comment:
I will look into a better way of keeping the reference to the "parent" object maintained. Should have an easy solution we don't know yet.